8.8/vips.modules

```diff
@@ -3,275 +3,251 @@
 <?xml-stylesheet type="text/xsl" href="moduleset.xsl"?>
 <moduleset>
 
-<!-- git repos -->
```

Comment:
Sorry for the auto-format changes. I should switch from this fancy IDE to VIM next time.